test/util_test.star

```diff
@@ -0,0 +1,56 @@
+util = import_module("/src/util.star")
+
+
+def test_label_from_image_short_image_name(plan):
+    image_name = "my-image"
+    image_label = util.label_from_image(image_name)
+
+    expect.eq(image_name, image_label)
+
+
+def test_label_from_image_63_character_image_name(plan):
+    image_name = ("tenletters" * 6) + "333"
+    expect.eq(len(image_name), 63)
+
+    image_label = util.label_from_image(image_name)
+    expect.eq(image_name, image_label)
+
+
+def test_label_from_image_64_character_image_name_no_slashes(plan):
+    image_name = ("tenletters" * 6) + "4444"
+    expect.eq(len(image_name), 64)
+
+    image_label = util.label_from_image(image_name)
+    expect.eq(image_name[-63:], image_label)
+
+
+def test_label_from_image_long_image_name_no_slashes(plan):
+    image_name = "tenletters" * 10
+    expect.eq(len(image_name), 100)
+
+    image_label = util.label_from_image(image_name)
+    expect.eq(image_name[-63:], image_label)
+
+
+def test_label_from_image_long_image_name_one_slash(plan):
+    image_suffix = "444"
+    image_name = "/".join(("tenletters" * 6, image_suffix))
```

Comment:
out of curiosity, any reason to join a tuple instead of a list? (here and below)
I personally find it slightly less immediately readable (due to the contiguous parentheses) but maybe that's just me.

Reply:
absolutely none, switched to lists!